test/integration/targets/openssl_certificate/tasks/ownca.yml

```
@@ -438,4 +438,123 @@
   when: select_crypto_backend != 'pyopenssl'
   register: ownca_authority_key_identifier_5
 
+- name: (OwnCA, {{select_crypto_backend}}) Ed25519 and Ed448 tests (for cryptography >= 2.6)
+  block:
+    - name: (OwnCA, {{select_crypto_backend}}) Generate privatekeys
+      openssl_privatekey:
+        path: '{{ output_dir }}/privatekey_{{ item }}.pem'
+        type: '{{ item }}'
```

```suggestion
        type: '{{ item }}'
        cipher: auto
        passphrase: Test123
```
to be pedantic :) regarding the playbooks from https://github.com/ansible/ansible/issues/59039

---

It needs to be further below, when creating the private key for the Own CA. I've added it there.